backend/src/v2/compiler/argocompiler/argo.go

```diff
@@ -117,7 +117,7 @@ func Compile(jobArg *pipelinespec.PipelineJob, kubernetesSpecArg *pipelinespec.S
 		templates: make(map[string]*wfapi.Template),
 		// TODO(chensun): release process and update the images.
 		driverImage:   "gcr.io/ml-pipeline/kfp-driver@sha256:8e60086b04d92b657898a310ca9757631d58547e76bbbb8bfc376d654bef1707",
```

Comment:
Not sure do you also want to parameterize the driver image? This image is also hard coded into the spec unless the Argo compiler move it to use Argo [HTTP Template](https://argoproj.github.io/argo-workflows/http-template/).